changelog.d/19716_doc_vrl_get_timezone_name.enhancement.md

```diff
@@ -0,0 +1,3 @@
+New VRL function `get_timezone_name` to return the configured/resolved IANA timezone name.
```

Comment:
Do we need this fragment?

Reply:
Ah, no, we don't since it'll pull in the fragment from VRL.

Reply:
no problem, file deleted